### DISCUSSION TITLE
中央通識修課經驗-自然語言處理

### DISCUSSION THREAD

**Discussion by Cerosop** [1101-GS4719-C/Python-Community-GS4719-C#1782] [General]
自然語言處理-李厚均

這堂課前半段在教基礎python語法，後半段在用python套件去實現自然語言處理功能，這堂課不會點名，但如果還不會自然語言處理的套件的話去上課一定不會吃虧，老師教得很仔細教材也很用心，整學期只有一次期末報告跟期中期末考，報告要求很簡單，但考試的題目蠻細的，不過還是比這堂課好一點，不過如果你神通廣大拿到考古，他有96%都是一樣的，最終我成績為99分，很涼很甜又可以學到東西。

**Comment by sorn32** [comment]
不確定有沒有變，但之前修他的圖像辨識的企業應用也差不多喔 σ`∀´)σ

**Comment by alvin119** [comment]
這學期我有修同樣老師的圖像辨識的企業應用，整體來說大同小異~
如果沒有拿到考古題的話其實題目沒有很簡單(也可能是我太混)，需要認真讀一下。
如果有考古題的話，記得期末考時也要回去複習期中考的考卷，期末考考題可能會出。

**Comment by asd99263fg** [comment]
聽說這個是我們學分學成要修的課程喔，感覺還不錯欸，不過那麼多的技巧是要自己學的話還是要用點心，不果整理來說看起來有考古會很吃香，但還是要自己讀啦，感覺這些課程不難但就是要自己去面對跟調整

**Comment by JustAHedgehog** [comment]
分享個人經驗，自己是大二時選修的課程，雖說是通識課，但老師教的內容都是滿滿的乾貨，包含Numpy, matplotlib, Pandas等等基礎函式庫也在課程中，但相對我們這門Pthon課程難度相差甚大，內容會更加資工，有興趣的可以挑戰看看，雖然很難，但可以從這門課較深度地了解AI的構造
